scripts/utils.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+die() { set +v; echo "$*" 1>&2 ; exit 1; }
+
+add_arg() {
+  # Helper for process_cells to build argument list.
```

Comment:
perhaps rename to `add_CLI_ARGS`... since the definition is farther away, we should be more clear about what it does.